contribs/gmf/src/directives/partials/layertree.html

```diff
@@ -48,3 +50,17 @@
       ngeo-layertree-listenersexpr="layertreeCtrl.listenersExpr">
   </li>
 </ul>
+<ul id="layer-group-{{::layertreeCtrl.uid}}"
+    ng-if="::layertreeCtrl.node.children && layertreeCtrl.isRoot"
+    ng-class="{in : layertreeCtrl.node.metadata.isExpanded}"
+    ngeo-sortable="layertreeCtrl.node.children"
```

Comment:
`ngeo-sortable` cannot handle a null array ?
Really would not prefer to duplicate this block if possible.